dev/tests/functional/tests/app/Magento/Catalog/Test/Block/Adminhtml/Product/Edit/Section/Options.php

```diff
@@ -402,4 +402,21 @@ public function getFileOptionElements()
     {
         return $this->_rootElement->getElements($this->hintMessage);
     }
+
+    /**
+     * @inheritdoc
+     */
+    protected function _fill(array $fields, SimpleElement $element = null)
```

Comment:
Could you provide more information about this change, please? I'm not sure it should be included as a part of the backport. Correct me if I'm wrong. Thank you